supabase/functions/alerts/alert_types/data_movement_stalled.ts

```diff
@@ -1,5 +1,5 @@
 import { AlertRecord, EmailConfig } from "../index.ts";
-import { isFinite } from "https://cdn.skypack.dev/lodash";
+import { isFinite } from "https://esm.sh/lodash?target=deno";
```

Comment:
`skypack` is made specially for browsers. So I think `esm.sh` is probably a bit "safer" of a choice.